### PR TITLE
blast support

### PR DESCRIPTION
**Pull request by dereknelson**
Fixes APP-1193

## What changed (plus any additional context for devs)
we have a new network again

## Screen recordings / screenshots
soon, my build is bricked but wanted to get this out 

## What to test
sends in blast!

### CODE REVIEW

File: src/networks/blast.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import { gasUtils } from '@/utils';
+import type { Chain } from '@wagmi/chains';
+// import { blast } from '@wagmi/chains';
+import { getAvalancheGasPrices, getBlastGasPrices } from '@/redux/gas';
+import { getRemoteConfig } from '@/model/remoteConfig';
+import { BLAST_MAINNET_RPC } from 'react-native-dotenv';
+import { BLAST_ETH_ADDRESS } from '@/references';
+
+const BLAST_CHAIN_ID = 81457;
+export const chainBlast: Chain = {
```
Comment (skylarbarrera) [inline src/networks/blast.ts:12]: can we migrate from `wagmi/chains` and use `viem/chains` looks like they moved it over and thats where the updates are being put now then we dont have to define this shiz ourself

Comment (dereknelson) [reply src/networks/blast.ts:12]: yeah

File: src/networks/blast.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      txHistory: true,
+      flashbots: false,
+      walletconnect: true,
+      swaps: false,
+      nfts: true,
+      pools: false,
+      txs: blast_tx_enabled,
+    },
+
+    gas: {
```
Comment (skylarbarrera) [inline src/networks/blast.ts:47]: supposedly blast is an optimism chain? we need to add the flag here so we add in the xtra gas prices

Comment (skylarbarrera) [reply src/networks/blast.ts:47]: actually jk idk where i heard this

**Review by skylarbarrera — APPROVED** [review]

**Pull request merged**